src/Event/EventDispatcherFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace GrumPHP\Event;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+/**
+ * @internal
+ */
+final class EventDispatcherFactory
```

Comment:
I bet it is temporary as well, when we would drop some old Sf, we could drop this class as well.
could you please add a TODO to do it after bumping requirement to `symfony/X: ^y.z` ?